### DISCUSSION TITLE
Fonoster Beta Program

### DISCUSSION THREAD

**Discussion by psanders** [fonoster/yaps#695] [Announcements]
<img alt="beta_program" src="https://github.com/user-attachments/assets/aa85ce27-7ac3-4a89-b55a-3ca6bf4fb6b4" />

We are inviting a select group to join our Fonoster Beta Program. As a participant, you'll gain early access to new features, help shape our platform's future, and receive direct support from our core team.

Please make sure you meet the criteria below to participate.

## Eligibility Requirements

- Your customers are based in 🇺🇸 🇨🇦 🇩🇴 🇦🇺
- Active Pro subscription 🏋️
- Comfortable with command-line tools
   - With don't have a dashboard yet❗
- Using Twilio or Onemax for trunking ☎️
- Willing to collaborate in private a channel 🔒

> More providers will be added soon.

## Join in three steps

1. Apply to the [beta program](https://dub.sh/HiXjqwc)
2. Upon acceptance:
   - Subscribe to the [Pro plan](https://dub.sh/pAcdfjr) <br /><br /><img width="300" alt="paymentlink" src="https://github.com/user-attachments/assets/785c02dd-3d6a-4cfc-a3ab-4fe922b21956" />
   - Join the private collaboration channel (we will share a link)
3. Begin development with Fonoster

We will guide you through the process and work with you and your team to ensure success. We look forward to collaborating with you!

**Comment by MuhireE** [comment]
Hi All,

i need to understand how below subjects work from fonoster

![image](https://github.com/user-attachments/assets/b2cf5951-dcc2-4d79-960b-c15fc3f33009)
![Capture](https://github.com/user-attachments/assets/4f76b438-014e-44ef-90e7-9ff0bb6d327a)

**Reply by psanders** [reply]
@MuhireE Sure thing. Any specific questions?

**Comment by MuhireE** [comment]
I want to learn on how you cater the 3 first points: multitenancy
Easy deployment of PBX functionalities
Programmable voice applications.

I'm ready for a call in case you have time.

**Reply by psanders** [reply]
@MuhireE I updated the docs. Let me know if that helps. Also happy to jump on a quick call to clarify anything.

**Reply by MuhireE** [reply]
Thank you so much, let's have a call on this Friday.

**Comment by diegobejardelaguila** [comment]
Hi bro, Do I need to be only from the US or Canada? Can't you call customers from Latin America and Peru?

**Reply by psanders** [reply]
We could open to other countries as long as the customer is using Twilio for telephony.

**Reply by djh00t** [reply]
I know it might be adding layers of complication but if Fonoster is trying to be an alternative to Twilio shouldn’t you be focussed on non Twilio use cases?

**Reply by psanders** [reply]
Hi @djh00t

Fair question. We're supporting Twilio first to help folks who already have Twilio for trunking.

The tech itself can integrate with any provider, but the challenge is delivering this feature at scale.

For example, it's proving incredibly hard for us to explain to people how to connect SIP trunking (we are a small team.)

However, because Twilio has a really good APIs for trunking, we can automate the process and it only takes a few seconds to interconnect.

Again, it's not a limitation of Fonoster, but the Beta Program is focused on bringing value fast to as many users as possible and then continuing to work on more use cases over time.

**Comment by priyanshiranawat15** [comment]
Hi ,from India 
I am currently not using Twilio , can I still use Fonoster ??

**Reply by pedro-sanders-tc** [reply]
We are mostly focused on the countries above. The main reason is that we haven't tested our system with SIP providers outside this group. That said, we'd be happy to explore opening in India if we see traction in the region.

**Reply by mhaseebcp786** [reply]
Hi @pedro-sanders-tc ,
Will be great if you are considering this and looking forward.

**Comment by rafeyhusain** [comment]
I have installed docker fonoster on my laptop to test fonoster using TypeScript. Do I need to be part of the Beta Program as mentioned here https://docs.fonoster.com/quickstart?

I just want to see how I can call a fonoster virtual number hosted on my laptop localhost (127.0.0.1)

1. Using my mobile phone

2. Using another TypeScript code on the same laptop that is hosting fonoster

**Reply by pedro-sanders-tc** [reply]
Hi Syed,

If you are running an instance of Fonoster, you don't need to be part of the Beta Program.
